### PR TITLE
Add main branch and Improve build instructions

### PR DESCRIPTION
**Pull request by marionbarker**
This PR should be merged at the same time that the main branch becomes available
* Update the testflight link to point to the main branch
* Add some graphics and attempt to improve the clarity of the build instructions

### CODE REVIEW

File: docs/EN/operate/build.md [hunk 1]
```diff
@@ -45,17 +48,44 @@ If you do not have a `Trio App Group`:
...
 | Trio | XC org nightscout TEAMID trio | org.nightscout.TEAMID.trio |
 | Trio LiveActivity | - | org.nightscout.TEAMID.trio.LiveActivity |
 | Trio Watch | XC IDENTIFIER | org.nightscout.TEAMID.trio.watchkitapp |
 | Trio WatchKit Extension | XC IDENTIFIER | org.nightscout.TEAMID.trio.watchkitapp.watchkitextension |
 
+### Add Trio App Group to Identifiers
+
+Open the App IDs Identifier page for your Apple Developer Account:
+
+* https://developer.apple.com/account/resources/identifiers/list
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: The URL is not clickable in RTD. Maybe move the link into the above text?

```
Open the [App IDs Identifier page](https://developer.apple.com/account/resources/identifiers/list) for your Apple Developer Account.
```

File: docs/EN/operate/build.md [hunk 2]
```diff
@@ -342,7 +366,7 @@ If you have customizations in the submodules that conflict with the update, you
 
 * Read which submodule failed to `checkout`
 * Change directory to that module: `cd modulename`
-* Discard your changes: `git reset --hard`
+* Discard your changes: `git reset --hard;git clean -fd`
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: Add a space after `;` to improve readability

File: docs/EN/operate/build.md [hunk 3]
```diff
@@ -432,3 +461,45 @@ To open Xcode, type `xed .` in the Terminal.
 
 Return to [Open Terminal](#open-terminal).
 
+## Special Section on Trio App Group
+
+* If you have never built Trio before - just follow the instructions on this page: [return to top](#)
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: `[return to top](#)` doesn't seem to work as intended. Also line 467.

Screenshot 2024-08-14 at 10 02 34 AM

File: docs/EN/operate/build.md [hunk 4]
```diff
@@ -432,3 +461,45 @@ To open Xcode, type `xed .` in the Terminal.
...
+* If you have already built Trio and have added the Trio App Group: [return to top](#)
+* If you have already built Trio, but you did not add the Trio App Group (or are not sure), please read this section for hints and help
+
+Click link for [Mac Build after App Group Change](#mac-build-after-app-group-change).
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: Maybe update to:

Instructions for [GitHub](#browser-build-after-app-group-change) | [Mac-Xcode](#mac-build-after-app-group-change)

```
Instructions for [GitHub](#browser-build-after-app-group-change) | [Mac-Xcode](#mac-build-after-app-group-change)
```

File: docs/EN/operate/build.md [hunk 5]
```diff
@@ -432,3 +461,45 @@ To open Xcode, type `xed .` in the Terminal.
...
+
+Return to [Add Trio App Group to Identifiers](#add-trio-app-group-to-identifiers)
+
+### Mac Build after App Group Change
+
+The first time you build Trio after the change from Loop App Group to Trio App Group, you are likely to the see the build error in the first graphic.
+
+![build error after switching to trio app group](img/trio-error-after-app-group-change.png)
+
+You must be connected to the internet and must allow Xcode to connect to Apple to update your app group. Tap in the four locations (in order) indicated graphic below to open the Signing & Certificates. The app group may momentarily appear in red font, then will update to black font for the trio-app-group. Continue until you have examined three targets `FreeAPS`, `FreeAPSWatch` and `FreeAPSWatch Watchkit Extension`. When those targets show the correct App Group, press the build symbol again.
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: > Tap in the four locations (in order) indicated graphic below to open the Signing & Certificates.

Seems to be a typo in here somewhere.

**Review by MikePlante1 — COMMENTED** [review]
Thanks for this update! Looks great, but I commented a few minor suggestions. I built to [this page](https://oidocs-mikeplante1.readthedocs.io/en/add_main_branch/operate/build.html) in my RTD to review.

**Comment by marionbarker** [comment]
Excellent comments Mike - thanks much. I think I updated to resolve all of these.

**Review by MikePlante1 — APPROVED** [review]

**Pull request merged**